MANUAL.txt

```diff
@@ -1917,6 +1917,17 @@ Pandoc uses these variables when [creating a PDF] with a LaTeX engine.
         - heightrounded
         ...
 
+`hyperrefoptions`
+:   option for [`hyperref`] package, e.g. `linktoc=all`;
+    repeat for multiple options:
+
+        ---
+        hyperref:
```

Comment:
Is this supposed to say `hyperrefoptions`?

Reply:
Yes, I corrected it.